src/pages/HammingCode.vue

```diff
@@ -80,8 +131,9 @@ export default defineComponent({
     });
 
     const showCalulations = ref(false);
+    const showTable = ref(false);
 
-    const dataBitCalculation = ref(new DataBitCalculation(0));
+    const dataBitCalculation = ref(new DataBitCalculation(BigInt(0)));
```

Comment:
maybe we could use a urlRef here aswell?
Could clutter the url though, so I think it is fine either way

Reply:
I wonder, should I 
- add BigInt support to the urlRef
- or let the user pass a serializer/deserializer to the urlRef, sort of like `urlRef("data-bit-cat", v => BigInt(v), v => "" + v)`
- or add a fully blown arbitrary object serializer/deserializer

Reply:
i think for now we are good, but it definitely would be interesting for the future